src/app/shared/daterange/daterange-portal.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, Inject } from '@angular/core';
+import { PORTAL_DATA } from './daterange-injection';
+
+@Component({
+  selector: 'grove-datepicker-portal',
+  template: `<div class="mat-datepicker-content"
+    [style.width]="componentData.width + 'px'"
+    [style.height]="componentData.height + 'px'">
+    <h2>{{componentData.content}}</h2>
```

Comment:
date range component will show up inside an h2? is that what we want to force as a container? seems semantically not great